src/utilities.py

```diff
@@ -309,15 +309,20 @@ def singular(plural):
     # otherwise we just added an s on the end
     return plural[:-1]
 
-def list_players(roles=None):
+def list_players(roles=None, rolemap=None):
```

Comment:
If you're going to use this a keyword-only argument, please forcibly make it keyword-only, e.g. `def list_players(roles=None, *, rolemap=None):`